stable/fairwinds-insights/ci/test-values.yaml

```diff
@@ -65,7 +65,7 @@ resourcesRecommendationsCronjob:
   enabled: true
   schedules: []
 
-closeTicketsCronjob:
+updateTicketsCronjob:
```

Comment:
huh this isn't doing anything anymore--we should remove these values and replace them with the new format `cronjobs.update-tickets`

Reply:
Removed